Source/scrollrt.cpp

```diff
@@ -521,7 +517,7 @@ void DrawPlayer(const Surface &out, int pnum, Point tilePosition, Point targetBu
 	}
 
 	if (!sprite) {
-		Log("Drawing player {} \"{}\": NULL CelSprite", pnum, player._pName);
+		Log("Drawing player \"{}\": NULL CelSprite", player._pName);

```

Comment:
For this error message the player index could potentially still be useful? `std::distance(Players, &player)` could work as a substitute for `pnum` so it's still easy to identify which player had dodgy sprite data.

Reply:
Any reason not to use std::distance when ever we need the ID for a network messages or save state?

Reply:
For objects and items 99% of the time they're referenced based on position so the concept of a network id isn't really relevant. std::distance would work for the cases where the code currently uses the array index, I'd rather completely change over to position based lookups instead.